FirebaseAppCheck/Sources/Core/Errors/FIRAppCheckErrorUtil.m

```diff
@@ -118,6 +118,42 @@ + (NSError *)errorWithFailureReason:(NSString *)failureReason {
                      underlyingError:nil];
 }
 
+#pragma mark - App Attest
+
++ (NSError *)appAttestGenerateKeyFailedWithError:(NSError *)error {
+  NSString *failureReason = @"Failed to generate a new cryptographic key for use with the App "
+                            @"Attest service (`generateKeyWithCompletionHandler:`).";
+  return [self appCheckErrorWithCode:FIRAppCheckErrorCodeUnknown
```

Comment:
Should this and the others be something more descriptive than `FIRAppCheckErrorCodeUnknown`?

Reply:
Ideally they would be but adding new error codes would technically be public API changes. If it otherwise seems reasonable, I'm hoping this could make it into M139.

Reply:
For now I've added TODOs to each of the methods referencing #11967 to track the addition of new `FIRAppCheckErrorCode` enum values.